app/src/main/java/org/sopt/havit/ui/search/SearchActivity.kt

```diff
@@ -90,6 +90,47 @@ class SearchActivity : BaseBindingActivity<ActivitySearchBinding>(R.layout.activ
         binding.tvSearchCancel.setOnClickListener {
             finish()
         }
+        with(searchContentsAdapter) {
+            setItemSettingClickListener(object : SearchContentsAdapter.OnItemSettingClickListener {
+                override fun onSettingClick(v: View, data: ContentsSearchResponse.Data, pos: Int) {
+                    val removeItem: (Int) -> Unit = {
+                        searchContentsAdapter.notifyItemRemoved(it)
+                    }
+                    ContentsMoreFragment(data, removeItem, pos).show(
+                        supportFragmentManager,
+                        "setting"
+                    )
+                }
+
+            })
+            setItemClickListener(object : SearchContentsAdapter.OnItemClickListener {
+                override fun onClick(v: View, data: ContentsSearchResponse.Data) {
+
+                    var intent = Intent(this@SearchActivity, WebActivity::class.java).apply {
+                        putExtra("url", data.url)
+                        putExtra("isSeen", data.isSeen)
+                        putExtra("contentsId", data.id)
+                    }
+                    startActivity(intent)
+                }
+            })
+            setItemHavitClickListener(object : SearchContentsAdapter.OnItemHavitClickListener {
+                override fun onHavitClick(
+                    v: View,
+                    data: ContentsSearchResponse.Data,
+                    isSeen: String
+                ) {
+                    searchViewModel.setIsSeen(data.id)
+                    if (isSeen == "seen") {
+                        searchViewModel.setHavitToast(false)
+                    } else {
+                        searchViewModel.setHavitToast(true)
+                    }
```

Comment:
```suggestion
                    searchViewModel.setHavitToast( if (isSeen == "seen") false else true ) 
```
이건 제 취향이긴 합니다만~ㅎㅎㅎ 참고만 해주세요!!

Reply:
이 코드가 되는진 모르겠지만 효림언니 취향 코드에서 더 줄이면
```suggestion
searchViewModel.setHavitToast( isSeen != "seen")
```
이런식으로도 가능할지......? 는 모르겠네용